A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -920,6 +958,22 @@ var A320_Neo_UpperECAM;
                             return (this.getCachedSimVar("L:AIRLINER_FLIGHT_PHASE", "Enum") == 6) && (this.getCachedSimVar("RADIO HEIGHT", "feet") < 800);
                         }
                     },
+                    {
+                        message: "AP OFF",
+                        style: "fail-3",
+                        important: true,
+                        isActive: () => {
+                            return (this.getAPWarning("AP", 4) == true);
```

Comment:
```js
if (this.getCachedSimVar('AUTOPILOT MASTER', 'Bool')) {
  SimVar.SetSimVarValue('L:A32NX_AP_LAST_DC_TIME', 'Number', 0);
  return false;
}
const last = SimVar.GetSimVarValue('L:A32NX_AP_LAST_DC_TIME', 'Number');
if (last > 0) {
  const now = Date.now();
  return (last - now) < 4000;
}
SimVar.SetSimVarValue('L:A32NX_AP_LAST_DC_TIME', 'Number', Date.now());
return true;
```

Reply:
I rebuild code more simply. is it okay working on isActive?